vue-app/src/router/index.ts

```diff
@@ -183,7 +183,7 @@ const routes = [
     },
   },
   {
-    path: '/transaction-success/:type/:hash',
+    path: '/transaction-success/:type/:hash?',
```

Comment:
👍

Reply:
Yes, I forgot to mention why I did this.

The reason is that when we are facing a partial contribution and the user refreshes the page, we lose that hash. So, for that case, I'm not showing any hash in the success screen.